File: src/main/java/ui/pageobjects/pages/VideoPlayerPage.java
package ui.pageobjects.pages;

import com.microsoft.playwright.Locator;
import com.microsoft.playwright.Page;

public class VideoPlayerPage extends BasePage {

    public VideoPlayerPage(Page page) {
        super(page);
    }

    public Locator playerWrapper() {
        return this.page.locator(".player_area");
    }

    public Locator videoTitle() {
        return this.page.locator("main h1");
    }
}

File: src/main/kotlin/ui/pageobjects/pages/VideoPlayerPage.kt
package ui.pageobjects.pages

import com.microsoft.playwright.Locator
import com.microsoft.playwright.Page

class VideoPlayerPage(page: Page?) : BasePage(page!!) {
    fun playerWrapper(): Locator {
        return page.locator(".player_area")
    }

    fun videoTitle(): Locator {
        return page.locator("main h1")
    }
}
